server/src/test/scala/services/PortStateSpec.scala

```diff
@@ -47,84 +47,6 @@ class PortStateSpec extends CrunchTestLike {
     success
   }
 
-  "Given a PortState with no purgable entries " +
```

Comment:
why this test not relevant any more ?

Reply:
Good question. The answer is that it's testing a function `purgeOlderThanDate` which isn't used anymore. So this PR also removes that function